packages/react/src/components/UIShell/SideNav.js

```diff
@@ -187,6 +197,12 @@ SideNav.propTypes = {
    */
   expanded: PropTypes.bool,
 
+  /**
+   * Provide the `href` to the id of the element on your package that is the
+   * main content.
+   */
+  href: PropTypes.string,
```

Comment:
could we make this `required` if we add a default or would that be a breaking change @tay1orjones ? SkipToContent has it as required with default 
<img width="752" alt="image" src="https://user-images.githubusercontent.com/40550942/234620143-e2e722d9-31a8-428d-b22f-131f6027f22e.png">

Reply:
I added the default prop and set the href as required.

Making the href attribute required for this component create any issues for developers who are already using it?
I don't know if make sense what I'm thinking

Reply:
The only problem I could see is that if someone pulls in this change, this href is going to be specified even if it wasn't previously. Their router solution may pick up on this or not, but I think it's possible it could cause unexpected side effects. 

We could err on the side of caution and make this opt-in for now. We could wait to mark it as required in v12. This could either be behind a feature flag or we could simply open an issue and wait to mark it required until v12 lands.

Reply:
> this href is going to be specified even if it wasn't previously
> we could simply open an issue and wait to mark it required until v12 lands.

@tay1orjones  If we make it not required won't it be specified anyway since we added a default? or you're saying we should remove the default as well? 
<img width="494" alt="image" src="https://user-images.githubusercontent.com/40550942/235251052-02412b50-85bb-4a51-afa6-80c09ddefa2f.png">

Reply:
Yeah, I think it needs to be optional and the default removed

Reply:
Perfect! I just removed the default also